src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/SpecialFileProviders/AppDesignerFolderSpecialFileProvider.cs

```diff
@@ -30,17 +28,12 @@ public AppDesignerFolderSpecialFileProvider(Lazy<IPhysicalProjectTree> projectTr
             _properties = properties;
         }
 
-        // For unit tests
-        protected AppDesignerFolderSpecialFileProvider()
-        {
-        }
```

Comment:
Removed unused constructor, as it gave warnings about non-null fields not being initialised.

Reply:
Turns out this is needed.